bigtable/google/cloud/bigtable/instance.py

```diff
@@ -28,6 +27,11 @@
 from google.cloud.bigtable.enums import RoutingPolicyType
 
 
+DEFAULT_SERVE_NODES = 3
```

Comment:
Please remove `DEFAULT_SERVE_NODES`.  It's going to be backwards incompatible, but the client is still in alpha.  I'd rather the user explicitly select the number of nodes rather than have a default of any sort.